playground/now-deployments/state-machines-gcp-eu/package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I moved the now deployment related stuff into those subfolders. This is kind of necessary because of Now project linking: https://zeit.co/docs/now-cli#commands/now/project-linking

I think it's actually better like this, as it decouples the Now setup from the app a bit more.